packages/horned_worm/horned_worm.0.3.6/opam

```diff
@@ -0,0 +1,31 @@
+opam-version: "2.0"
+synopsis: "An easy functional Web app micro framework"
+description: """
+An easy functional Web app micro framework.
+You can create REST API server with composable functions.
+"""
+maintainer: "obiwanko@me.com"
+authors: ["Kazuo Koga"]
+license: "MIT"
+homepage: "https://github.com/kkazuo/horned_worm"
+bug-reports: "https://github.com/kkazuo/horned_worm/issues"
+dev-repo: "git+https://github.com/kkazuo/horned_worm.git"
+depends: [
+  "ocaml"           {>= "4.03.0"}
+  "async"           {>= "v0.11.0" & < "v0.13.0"}
+  "cohttp-async"    {>= "1.0.2" & < "3.0.0"}
+  "logs"            {>= "0.6.2"}
+  "re"              {>= "1.7.3"}
+  "yojson"          {>= "1.6.0"}
+  "dune"            {build & >= "1.0.1"}
```

Comment:
```suggestion
  "dune"            {>= "1.0.1"}
```

Dune cannot be tagged as build dependency due to forward compatibility issues (see https://github.com/ocaml/opam-repository/issues/14185)

Reply:
Can you upstream the change?